src/js/view/view.js

```diff
@@ -1064,11 +1054,6 @@ define([
             _stateHandler(model, model.get('state'));
         }
 
-        function _resizeAspectModeCaptions() {
```

Comment:
When not rendering natively, we need this in order to position captions correctly for aspect mode players. We can get the container height off the model instead, though.